src/plugins/intel_cpu/src/nodes/stft.cpp

```diff
@@ -73,15 +77,103 @@ bool STFT::created() const {
     return getType() == Type::STFT;
 }
 
+static void transpose_out(const char* in,
+                          char* out,
+                          const VectorDims& in_shape,
+                          const std::vector<size_t>& axes_order,
+                          const VectorDims& out_shape,
+                          size_t elem_size) {
+    parallel_for4d(out_shape[0],
+                   out_shape[1],
+                   out_shape[2],
+                   out_shape[3],
+                   [in, out, axes_order, &in_shape, &out_shape, elem_size](size_t i, size_t j, size_t k, size_t l) {
+                       size_t in_indexes[4];
+                       in_indexes[axes_order[0]] = i;
+                       in_indexes[axes_order[1]] = j;
+                       in_indexes[axes_order[2]] = k;
+                       in_indexes[axes_order[3]] = l;
+                       size_t in_off =
+                           ((in_indexes[0] * in_shape[1] + in_indexes[1]) * in_shape[2] + in_indexes[2]) * in_shape[3] +
+                           in_indexes[3];
+                       size_t out_off = ((i * out_shape[1] + j) * out_shape[2] + k) * out_shape[3] + l;
+                       cpu_memcpy(out + out_off * elem_size, in + in_off * elem_size, elem_size);
```

Comment:
1. Good point, thanks! I've optimized this transpose logic to copy `dim[3] * elem_size` at once.
2. I've considered usage of the Transpose executor, but not all of the attrs needed to create it were clear for me, thus I had some issues. It can be still considered as possible further improvement.

Reply:
It should be also possible to avoid the final transpose, and produce transposed output by modification of the stft loop.
Exploration of the transpose optimization can be considered as a follow up topic.